src/definitions/constants.rs

```diff
@@ -14,6 +14,11 @@ pub(crate) const N_DEFAULT_TOPICS: usize = 1; // Events have one default topic.
 pub(crate) const CONSUMED_MSG_TO_L2_ENCODED_DATA_SIZE: usize =
     (L1_TO_L2_MSG_HEADER_SIZE + 1) - CONSUMED_MSG_TO_L2_N_TOPICS;
 
+lazy_static! {
+    pub(crate) static ref VERSION_QUERY: Felt252 =
```

Comment:
the name in cairo-lang is VERSION_QUERY_BASE I think we should have the same name because VERSION_QUERY has an specific meaning in this context:

`QUERY_VERSION = QUERY_VERSION_BASE + TRANSACTION_VERSION`
Where TRANSACTION_VERSION = 1